src/test/java/com/revature/maadcars/services/UserServiceTest.java

```diff
@@ -0,0 +1,82 @@
+package com.revature.maadcars.services;
+
+import com.revature.maadcars.controllers.UserController;
+import com.revature.maadcars.models.User;
+import com.revature.maadcars.repository.UserRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.when;
+
+@SpringBootTest
```

Comment:
@SpringBootTest and UserController aren't needed for service-layer unit test, should remove all references to either.